src/admin/users/views/UserOverview.tsx

```diff
@@ -648,10 +656,15 @@ const UserOverview: FunctionComponent<UserOverviewProps & RouteComponentProps &
 				return !isNil(lastAccessDate) ? formatDate(lastAccessDate) : '-';
 
 			case 'temp_access':
-				const tempAccess = get(rowData, 'user.temp_access');
-				const hasTempAccess = get(tempAccess, 'from') && get(tempAccess, 'until');
-
-				return hasTempAccess && !isBlocked ? 'Ja' : 'Nee';
+				const tempAccess = get(rowData, 'user.temp_access.current.status');
+				switch (tempAccess) {
+					case 0:
+						return t('admin/users/views/user-overview___tijdelijke-toegang-nee');
+					case 1:
+						return t('admin/users/views/user-overview___tijdelijke-toegang-ja');
+					default:
+						return '';
```

Comment:
Mss net als de andere cols een `-` returnen als er geen andere values zijn?
Dat zie ik hier toch vaker terugkomen, tenzij anders afgesproken.

Reply:
Makes sense. Dat is overal waar er geen waarde is zo. Fixed.